### PR TITLE
Update evm-core to 0.26.1

### PR DESCRIPTION
**Pull request by joshuajbouw**
This is an emergency Sputnik EVM update as per [RUSTSEC-2021-0066](https://rustsec.org/advisories/RUSTSEC-2021-0066.html).

### CODE REVIEW

**Review by artob — APPROVED** [review]

File: Cargo.toml [hunk 1]
```diff
@@ -41,7 +41,7 @@ rpath = false
 blake2 = { git = "https://github.com/near/near-blake2.git", version = "0.9.1", default-features = false }
 borsh = { version = "0.8.2", default-features = false }
 bn = { package = "aurora-bn", git = "https://github.com/aurora-is-near/aurora-bn.git", default-features = false }
-evm = { git = "https://github.com/aurora-is-near/sputnikvm", rev = "2a8a3e9", default-features = false }
+evm = { version = "0.26.0", default-features = false }
```
Comment (mfornet) [inline Cargo.toml:44]: I think we should use 0.26.1?

**Review by mfornet — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (mfornet) [reply Cargo.toml:44]: Screenshot 2021-05-12 at 12 37 10

Comment (joshuajbouw) [reply Cargo.toml:44]: No, thats for `evm-core` not `evm`.

Comment (artob) [reply Cargo.toml:44]: https://github.com/aurora-is-near/aurora-engine/commit/a51008afb54e0ff467dc1ed8592116891a47f4b2#r50714346

**Pull request merged**